recipes/serd/all/conandata.yml

```diff
@@ -0,0 +1,8 @@
+sources:
+  "0.30.12":
+    serd:
+      url: "https://gitlab.com/drobilla/serd/-/archive/v0.30.12/serd-v0.30.12.tar.gz"
+      sha256: "dd218aabc7df91f21b7c5ee8cea2604b7a88a4fb4d853215d3d1183d6749f19c"
+    autowaf:
+      url: "https://gitlab.com/drobilla/autowaf/-/archive/aeef9f5fdf416d9b68c61c75de7dae409f1ac6a4/autowaf-aeef9f5fdf416d9b68c61c75de7dae409f1ac6a4.tar.gz"
```

Comment:
I will double check

Reply:
No. autowaf is used throughout https://gitlab.com/drobilla/serd/-/blob/master/wscript for most important configs. Replacing it would basically mean to re-implement the build.

Reply:
...but probably we can package `autowaf` as well, right? I'm ok if it is not for this PR, but for a future one, but maybe a `TODO` here would help to remember about it.